instrumentation/jdbi3/src/main/java/brave/jdbi3/Jdbi3BravePlugin.java

```diff
@@ -0,0 +1,173 @@
+/*
+ * Copyright The OpenZipkin Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+package brave.jdbi3;
+
+import java.net.URI;
+import java.net.URISyntaxException;
+import java.sql.Connection;
+import java.sql.SQLException;
+import java.time.Instant;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+import org.jdbi.v3.core.Jdbi;
+import org.jdbi.v3.core.spi.JdbiPlugin;
+import org.jdbi.v3.core.statement.SqlStatements;
+import org.jdbi.v3.core.statement.StatementContext;
+import org.jdbi.v3.core.statement.StatementContextListener;
+
+import brave.Span;
+import brave.Tracing;
+
+/**
+ * Instrumentation for JDBI 3.
+ * <p>
+ * JDBI plugin that sends a trace span for every SQL command executed by Jdbi.
```

Comment:
This sounds redundant. A lot of code here has had a lot of review, so you can avoid back and forth by copying it first, then revising. For example, this says what (a couple times) not why. e.g. if you look at the reference I mentioned, the javadoc says this `A MySQL query interceptor that will report to Zipkin how long each query takes` So the why is about the query. Also, we don't need to get specific to the version as that's already in the package name.

https://github.com/openzipkin/brave/blob/master/instrumentation/mysql8/src/main/java/brave/mysql8/TracingQueryInterceptor.java#L23-L31